funcframework/framework_test.go

```diff
@@ -586,49 +609,55 @@ func TestCloudEventFunction(t *testing.T) {
 
 func TestDeclarativeFunctionHTTP(t *testing.T) {
 	funcName := "httpfunc"
+	funcResp := "Hello World!"
 	os.Setenv("FUNCTION_TARGET", funcName)
+	defer os.Unsetenv("FUNCTION_TARGET")
 
-	if err := RegisterHTTPFunctionContext(context.Background(), "/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think we still want to have some test that mixes both the older RegisterHTTPFunctionContext and `functions.HTTP` calls and makes sure things work as expected.

Reply:
Done